### PR TITLE
Fail missing args non dev

### LINKED ISSUE #2137: fail if env not dev and pcr2 not passed to prepare or authorize

### PR DESCRIPTION
**Pull request by Sneagan**
### Summary

Feature/fail missing args non dev. closes #2137

### Type of Change

- [x] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before Requesting Review

- [x] Does your code build cleanly without any errors or warnings?
- [x] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security and/or privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

File: tools/payments/cmd/authorize/main.go [hunk 1]
```diff
@@ -90,6 +90,10 @@ func main() {
 		log.Printf("Redis: %s, %s\n", *redisAddr, *redisUser)
 	}
 
+	if *env != "dev" && len(*pcr2) != 96 {
```
Comment (pavelbrm) [inline tools/payments/cmd/authorize/main.go:93]: Should `env` here and below be consistent with other places?

Comment (Sneagan) [reply tools/payments/cmd/authorize/main.go]: When passing the `-e` flag to the tooling we've been using the abbreviated `dev`. However, this is probably a good place to enforce a better practice. Fixed.

File: tools/payments/cmd/prepare/main.go [hunk 2]
```diff
@@ -88,6 +88,10 @@ func main() {
 		log.Printf("Operator Key File Location: %s\n", *key)
 	}
 
+	if *env != "dev" && len(*pcr2) != 96 {
```
Comment (pavelbrm) [inline tools/payments/cmd/prepare/main.go:91]: This is still left. 😄

Comment (Sneagan) [reply tools/payments/cmd/prepare/main.go]: How embarrassing. XD
Fixed.

**Review by husobee — APPROVED** [review]

**Pull request merged**